src/views/InteractionDesigner.vue

```diff
@@ -251,7 +251,10 @@ export class InteractionDesigner extends mixins(Lang, Routes) {
     (global as any).builder = this
 
     // Listen to Flow updates to maintain "hasFlowChanges" state
-    this.$store.subscribe(this.handleFlowChanges.bind(this))
+    // 1- Listen to mutations first
+    this.$store.subscribe(this.handleFlowChangesViaMutations.bind(this))
```

Comment:
We created `flow_removeBlock` mutation to make it work with undo/redo feature; I'd like to suggest picking that change instead of creating another event handler to keep things simple.

https://github.com/FLOIP/flow-builder/pull/472/files#diff-eea673c657b7edcdbf00ab176fcc8ec0a84d21e8a8dfa9ff8a014c7510a144a0R151-R160

The other commit https://github.com/FLOIP/flow-builder/commit/464073a0bf10de40521cbfe4249e1eb2d7b920e4

Reply:
Shall we decline this PR and wait for undo/redo instead?

Reply:
If we cannot cherry pick commits from the undo/redo, then just replicate the solution here. Would be great to not have this 2nd binding